docs/_includes/star-rating.html

```diff
@@ -15,12 +17,80 @@
     {% endfor %}
     </ul>
 
-    <p>Below is the code example for how to use the "star-rating-2-5" symbol. Please refer to the <a href="#icon">icon section</a> for full guidance on how to include and use SVG symbols.
+    <p>Below is the code example for how to use the "star-rating-2-5" symbol. Please refer to the <a href="#icon">icon section</a> for full guidance on how to include and use SVG symbols.</p>
 
     {% highlight html %}
 <svg role="img" aria-label="Rating: 2.5/5" class="star-rating" focusable="false" height="12">
     <use xlink:href="#star-rating-2-5"></use>
 </svg>
     {% endhighlight %}
 
+    <h3>Method 2: Independent Stars</h3>
+    <p>This is built using fine-grained pieces (individual stars) for optimal flexibility and for allowing potential independant star treatments (i.e. animations).</p>
+    <p>
+        <ul class="demo-icon-boxes">
+            {% assign ratings = "0, 0-5, 1, 1-5, 2, 2-5, 3, 3-5, 4, 4-5, 5" | split: ", " %}
+            {% for rating in ratings %}
+            <li>
+                <span>{{ rating }}</span>
+                <div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="{{ rating }}">
+                    <svg class="star-rating-icon" aria-hidden="true" focusable="false">
+                        <use class="star-rating-icon__star-full" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
```

Comment:
We have a jekyll "mixin" or something like that which you can use to load the icons. 
https://github.com/eBay/skin/blob/master/docs/_includes/program-badge.html#L11-L13
Do something like this

Reply:
Yea, I actually tried doing that at the very beginning of the work and had issues, but I was originally using `svg` masks and it looked to be an issue. I'll try it again since I changed methods and am using clipping now.

Reply:
Ah, I just realized another reason why I switched to doing this instead of using `{% include symbol.html name=star-full  %}`. I need classes on the `use` elements and symbol doesn't currently support that. I suppose I could have extended symbol with that feature but want to avoid scope creep.

Reply:
I think we should do it anyways. It's literally a simple change like this:
```
--- a/docs/_includes/symbol.html
+++ b/docs/_includes/symbol.html
@@ -1,7 +1,7 @@
 {% if include.type == "program-badge" %}
 <use xlink:href="{{ page.static_dir }}/common/program-badges.svg#program-badge-{{ include.name }}"></use>
 {% elsif include.type == "star-rating" %}
-<use xlink:href="{{ page.static_dir }}/common/star-rating.svg#star-rating-{{ include.name }}"></use>
+<use class="{{ include.className }}" xlink:href="{{ page.static_dir }}/common/star-rating.svg#star-rating-{{ include.name }}"></use>
 {% else %}
 <use xlink:href="{{ page.static_dir }}/common/icons.svg#icon-{{ include.name }}{{ include.postfix }}"></use>
 {% endif %}
```

Just add `class="{{ include.className }}"` and it should be fine.
It's not scope creep since the code is relying on classes and the mixin should be updated accordingly.

Reply:
Ok, I'll do it. That specific example wouldn't work (star-rating is for full star sets) but I get the picture.